packages/flutter_tools/lib/src/android/android_studio.dart

```diff
@@ -66,14 +66,28 @@ class AndroidStudio implements Comparable<AndroidStudio> {
         pathsSelectorValue = jvmProperties['idea.paths.selector'] as String;
       }
     }
-    final String presetPluginsPath = pathsSelectorValue == null
-      ? null
-      : globals.fs.path.join(
-        globals.fsUtils.homeDirPath,
-        'Library',
-        'Application Support',
-        pathsSelectorValue,
-      );
+
+    final int major = version?.major;
+    final int minor = version?.minor;
+    String presetPluginsPath;
+    if (pathsSelectorValue != null) {
+      if (major >= 4 && minor >= 1) {
```

Comment:
Note: this code does not account for possibility of `major` and `minor` being `null`. Which causes it to crash: https://github.com/flutter/flutter/issues/69863

Same below. 

(NNBD would have actually rejected this code)

Reply:
#69954 fixed